packages/components/src/components/link-tile-product/link-tile-product-styles.ts

```diff
@@ -0,0 +1,180 @@
+import { buildResponsiveStyles, getCss, Theme } from '../../utils';
+import {
+  addImportantToEachRule,
+  colorSchemeStyles,
+  getInsetJssStyle,
+  getThemedColors,
+  getTransition,
+  hostHiddenStyles,
+  hoverMediaQuery,
+  prefersColorSchemeDarkMediaQuery,
+} from '../../styles';
+import {
+  borderRadiusLarge,
+  borderRadiusMedium,
+  borderWidthBase,
+  fontLineHeight,
+  fontSizeTextXSmall,
+  fontWeightRegular,
+  getFocusStyle,
+  getMediaQueryMin,
+  headingSmallStyle,
+  spacingFluidMedium,
+  spacingFluidSmall,
+  spacingFluidXSmall,
+  textXSmallStyle,
+  textXXSmallStyle,
+  themeLightStateFocus,
+} from '@porsche-design-system/utilities-v2';
+import { BreakpointCustomizable } from '../../utils/breakpoint-customizable';
+import { headerSlot, LinkTileProductAspectRatio } from './link-tile-product-utils';
+
+const cssVariableMixBlendMode = '--p-link-tile-product-mix-blend-mode';
+
+export const getComponentCss = (
+  hasLikeButton: boolean,
+  hasSlottedAnchor: boolean,
+  aspectRatio: BreakpointCustomizable<LinkTileProductAspectRatio>,
+  theme: Theme
+): string => {
+  const { primaryColor, contrastHighColor, backgroundSurfaceColor } = getThemedColors(theme);
+  const {
+    primaryColor: primaryColorDark,
+    contrastHighColor: contrastHighColorDark,
+    backgroundSurfaceColor: backgroundSurfaceColorDark,
+  } = getThemedColors('dark');
+  return getCss({
+    '@global': {
+      ':host': {
+        display: 'block',
+        position: 'relative',
```

Comment:
Would it be possible to set `position: relative` on `.root` instead?

Reply:
Focus outline doesn't work then since its on the slotted anchor